art/art_param.py

```diff
@@ -129,7 +129,9 @@
     "yie_ar_k",
     "z-pilot",
     "zig_zag",
-    "zone7"]
+    "zone7",
+    "fancy75",
```

Comment:
Why you add `fancy75` and `fancy81` to `UPPERCASE_FONTS` list ? (it's for case that font **only** support capital letters)

Reply:
These fonts map lower-case and upper-case character to same upper-case one, isn't this the definition of UPPERCASE_FONTS?
maybe i have misunderstood with this.